### PR TITLE
New version: ArtifexSoftware.GhostScript version 10.02.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Pull request has been created with [WinGet Automation](https://github.com/vedantmgoyal2009/vedantmgoyal2009/tree/main/winget-pkgs-automation) :rocket:

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/a/ArtifexSoftware/GhostScript/10.02.0/ArtifexSoftware.GhostScript.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+PackageVersion: 10.02.0
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: uninstallPrevious
+ProductCode: GPL Ghostscript 10.02.0
+ReleaseDate: 2023-09-13
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/a/ArtifexSoftware/GhostScript/10.02.0/ArtifexSoftware.GhostScript.installer.yaml:14]:
```suggestion
- Architecture: x64
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

Is there an installer switch to complete the install automatically?

**Comment by b1n4ryj4n** [comment]
@stephengillie 

> Is there an installer switch to complete the install automatically?

No it is not. Since version 10.01.0 they disabled because of security reasons the silent install option (Source: https://artifex.com/news/ghostscript-10.01.0-disabling-silent-install-option).

But it is essential that people wont install version 10.0.0 as main version because < 10.02.0 have massive vulnerability that allows remote code execution. [CVE-2023-43115]

**Pull request closed without merging**